netbox/resource_netbox_vrf_test.go

```diff
@@ -94,6 +105,40 @@ resource "netbox_vrf" "test_tags" {
 	})
 }
 
+func TestAccNetboxVrf_tenant(t *testing.T) {
+
+	testSlug := "vrf_tenant"
+	testName := testAccGetTestName(testSlug)
+	resource.ParallelTest(t, resource.TestCase{
+		Providers: testAccProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetboxVrfTenantDependencies(testName) + fmt.Sprintf(`
+resource "netbox_vrf" "test_tenant" {
+  name = "%[1]s"
+  tenant_id = netbox_tenant.test_tenant_a.id
+}`, testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("netbox_vrf.test_tenant", "name", testName),
+					resource.TestCheckResourceAttrPair("netbox_vrf.test_tenant", "tenant_id", "netbox_tenant.test_tenant_a", "id"),
+				),
+			},
+			{
+				Config: testAccNetboxVrfTenantDependencies(testName) + fmt.Sprintf(`
+resource "netbox_vrf" "test_tenant" {
```

Comment:
see above

Reply:
> This look good if you fix the names of the vrf objects. I think canonical would be "test_vrf"

I'm not completely sure. I can do the change, of course it is just a string change, but I followed the style used in `func TestAccNetboxVrf_tags(t *testing.T) {` (line63 and following). (simply replaced tags with tenant)

Just let me know If I misinterpreted the situation

Reply:
yeah, I guess you could use "test_<FEATURE BEING TESTED>". In this case, I failed to see that there is an extra test case for tenant. Agree.